modules/ding_config_import/README.md

```diff
@@ -0,0 +1,74 @@
+# Ding configuration import
+This module can be used to import configuration changes into the site and enable/disable modules.
+
+The configuration is described by using YAML files with arrays that are flattened to variable keys
+if they are associative arrays. If "normal" arrays they are written as arrays to variables.
+
+If an associative array contains the stop key `array: 1` the array it is placed in is not flattened
+thereby allow for complex variable configuration.
+
+The module comes with an example file to help users getting started with the configuration files.
+
+## How to use
+Simply download the example file that comes with the module add edit it to set the different
+configuration variables.
+
+Upload the file at `/admin/config/ding/config` and clear the cache.
+
+## How it is parsed
+
+This yaml snippet will be transformed to the following variables as store in the database.
+```yaml
+---
+opensearch:
+  url: "https://opensearch.addi.dk/b3.5_5.2/"
```

Comment:
These examples are excellent. I wish all our READMEs could be like this :smile:.